### ISSUE TITLE
Issue about the dataset's split method

### ISSUE THREAD

**Issue by Asakurayoho** [dewenzeng/MCE_dataset#2]
Thank you for providing this public  MCE dataset. According to the description of the MCE dataset in the paper, can I think that the dataset is divided into the first 70 subjects (ID 000-069) with 700 images as the training set and the last 30 subjects (ID 070-099) with 300 images as the test set?
